CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 
 
-## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...master)
+## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1.1+1...master)
 
-_nothing noteworthy, yet_
+* Summary reporter getter is now thread-safe.
+  * [CODETOOLS-7903390](https://bugs.openjdk.org/browse/CODETOOLS-7903390)
+
+## [7.1.1+1](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...jtreg-7.1.1+1)
+
+* Summary reporter getter is now thread-safe.
+  * [CODETOOLS-7903390](https://bugs.openjdk.org/browse/CODETOOLS-7903390)
```

Comment:
Seems like duplicate entries.

Reply:
The intention was to document a backport.

As the upcoming `7.2` release will contain all fixes from older releases, I'll remove the duplicate entry.